FirebaseStorage/Sources/Storage.swift

```diff
@@ -133,39 +138,40 @@ import FirebaseAuthInterop
+   * - Returns: An instance of `StorageReference` referencing the root of the storage bucket.
    */
   @objc open func reference() -> StorageReference {
     return StorageReference(impl: impl.reference(), storage: self)
   }
 
   /**
-   * Creates a StorageReference given a gs:// or https:// URL pointing to a Firebase Storage
-   * location. For example, you can pass in an https:// download URL retrieved from
-   * [StorageReference downloadURLWithCompletion] or the gs:// URI from
-   * [StorageReference description].
-   * @param string A gs:// or https:// URL to initialize the reference with.
-   * @return An instance of StorageReference at the given child path.
-   * @throws Throws an exception if passed in URL is not associated with the FirebaseApp used to initialize
-   * this FirebaseStorage.
+   * Creates a StorageReference given a `gs://` or `https://` URL pointing to a Firebase Storage
+   * location. For example, you can pass in an `https://` download URL retrieved from
+   * `StorageReference.downloadURL(completion:)` or the `gs://` URL from
+   * `StorageReference.description`.
+   * - Parameter url A gs:// or https:// URL to initialize the reference with.
+   * - Returns: An instance of StorageReference at the given child path.
+   * - Throws: Throws a fatal error if `url` is not associated with the `FirebaseApp` used to initialize
+   *     this Storage instance.
    */
-  @objc open func reference(forURL string: String) -> StorageReference {
-    return StorageReference(impl: impl.reference(forURL: string), storage: self)
+  @objc open func reference(forURL url: String) -> StorageReference {
+    return StorageReference(impl: impl.reference(forURL: url), storage: self)
   }
 
   /**
-   * Creates a StorageReference initialized at a child Firebase Storage location.
-   * @param string A relative path from the root to initialize the reference with,
-   * for instance @"path/to/object".
-   * @return An instance of StorageReference at the given child path.
+   * Creates a `StorageReference` initialized at a location specified by the `path` parameter.
+   * - Parameter path A relative path from the root of the storage bucket,
+   *     for instance @"path/to/object".
+   * - Returns: An instance of `StorageReference` pointing to the given path.
    */
-  @objc(referenceWithPath:) open func reference(withPath string: String) -> StorageReference {
-    return StorageReference(impl: impl.reference(withPath: string), storage: self)
+  @objc(referenceWithPath:) open func reference(withPath path: String) -> StorageReference {
+    return StorageReference(impl: impl.reference(withPath: path), storage: self)
   }
 
   /**
-   * Configures the Storage SDK to use an emulated backend instead of the default remote backend.
+   * Configures the Storage SDK to use an emulated backend instead of the default remote backend. This method should be
+   * called before invoking any other methods on a new instance of `Storage`.
```

Comment:
optional but L168 looks a little long. Maybe:
```suggestion
   * Configures the Storage SDK to use an emulated backend instead of the default remote backend. 
   * This method should be called before invoking any other methods on a new instance of `Storage`.
```